Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM yastdevel/ruby:sle12-sp5
 
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
```

Comment:
I'd avoid `--gpg-auto-import-keys` it's not needed anymore, the Docker image has been changed so it already contains the used GPG keys.

```suggestion
RUN zypper --non-interactive in --no-recommends \
```

Reply:
Suggestion applied, thanks!